packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+
+export default class TextField extends React.Component<{||}> {
+    render(): React.Node {
+        return (
+            <View style={[styles.container, styles.default]}>
```

Comment:
**I'm on board with just having the `input` alone**. I actually did start off by just having the `input` alone without a `View` wrapper, but then I noticed on Figma that the TextField had a few design layers so I assumed that it meant it needed a wrapper `View` to apply those styles.

Reply:
Also, in terms of the "icon" or another element next to the `input`, I guess if that's something we might want later it would be better to have a separate component that derives from TextField that handles that.